esphome/components/abbaurora/text_sensor.py

```diff
@@ -0,0 +1,38 @@
+import esphome.codegen as cg
+import esphome.config_validation as cv
+from esphome.components import text_sensor
+from esphome.const import (
+    CONF_ID,
+)
+from . import CONF_ABBAURORA_ID, ABBAURORA_COMPONENT_SCHEMA
+
+CONF_INVERTER_VERSION = "inverter_version"
+CONF_CONNECTION_STATUS = "connection_status"
+CONF_IDENTIFICATION = "identification"
+AUTO_LOAD = ["abbaurora"]
```

Comment:
```suggestion
DEPENDENCIES = ["abbaurora"]
```

Since abbaurora contains required keys